### PR TITLE
WEBUI-721: fix document import tab

### PR DESCRIPTION
**Pull request by richardsd**

### CODE REVIEW

**Comment by richardsd** [comment]
webui-721

File: elements/document/nuxeo-document-import.js [hunk 1]
```diff
@@ -372,8 +372,6 @@ Polymer({
     <nuxeo-operation id="fileManagerImport" op="FileManager.Import" sync-indexing></nuxeo-operation>
     <nuxeo-document
       id="docRequest"
-      doc-path="[[targetPath]]"
```
Comment (Gabez0r) [inline elements/document/nuxeo-document-import.js:375]: Please note that the targetPath is also changed by `Nuxeo.DocumentCreationBehavior`.

Comment (richardsd) [reply elements/document/nuxeo-document-import.js:375]: It doesn't seem to have an effect in the import tab, thank you for the review 👍

File: elements/document/nuxeo-document-import.js [hunk 2]
```diff
@@ -1266,18 +1279,22 @@ Polymer({
           .forEach((index) => {
             this.splice('remoteFiles', index, 1);
           });
+        /*
+         * Prevent this._selectDoc(0) from storing the previously selected file,
```
Comment (Gabez0r) [inline elements/document/nuxeo-document-import.js]: nit: I would even put an "XXX" here, so we could find this later :P

**Review by Gabez0r — DISMISSED** [review]
Great job! 👏 💪

File: elements/document/nuxeo-document-import.js (hunk 2 above)
Comment (richardsd) [reply elements/document/nuxeo-document-import.js]: Let me add that indeed. Thanks 😄

**Review by Gabez0r — APPROVED** [review]

**Review by semisse — APPROVED** [review]

**Pull request merged**